src/components/SortableTable/SortableTable.tsx

```diff
@@ -1,28 +1,48 @@
-import React, { useMemo, useState } from "react";
+import { useMemo, useState, Fragment } from "react";
+import HTMLString from "react-html-string";
 import "./SortableTable.css";
 
-interface Column {
+interface header {
   key: string;
-  header: string;
+  value: string;
 }
 
+interface Stats {
+  mean: number;
+  std: number;
+}
+
+type HtmlString = string;
+
 interface TableData {
-  [key: string]: any;
+  [key: string]: Stats | number | string | HtmlString[];
 }
 
 interface SortState {
-  sortKey: string | null;
+  sortKey: string;
   sortOrder: "asc" | "desc";
 }
 
 interface SortableTableProps {
   data: TableData[];
-  columns: Column[];
+  verticalHeaders: header[];
+  horizontalHeaders?: header[];
+}
+
+interface TableCellContentProps {
+  content: Stats | number | string | HtmlString[];
 }
 
-const SortableTable: React.FC<SortableTableProps> = ({ data, columns }) => {
+const SortableTable = ({
+  data,
+  verticalHeaders = [],
+  horizontalHeaders = [],
+}: SortableTableProps) => {
   const [sortState, setSortState] = useState<SortState>({
-    sortKey: columns[0].key,
+    sortKey:
+      horizontalHeaders.length > 0
+        ? horizontalHeaders[0].key
+        : verticalHeaders[0].key,
```

Comment:
This means that the key should be unique in both headers. 
I dont see an example using `horizontalHeaders` does this work?

Reply:
I removed `horizontalHeaders`